### DISCUSSION TITLE
Trouble Indexing Attributes

### DISCUSSION THREAD

**Discussion by brooklynebkbk** [Pagefind/pagefind#393] [Q&A]
Hi! I have pagefind integrated on a small 11ty project. Everything seems to be working well except for adding HTML attributes to the search index with the `data-pagefind-index-attrs` attribute.

I've included `data-pagefind-index-attrs="title"` alongside other pagefind attributes and this pagefind attribute in particular does not seem to be indexing the referenced data as expected.

Appreciate any help - thanks!

**Comment by bglw** [comment]
Hi @brooklynebkbk — sorry for missing this! I didn't see it come through.

Do you have any example code you can share? I'd be happy to take a look!

If it helps to see an example in-situ, here is the test case for this feature indexing some alt text:

https://github.com/CloudCannon/pagefind/blob/f46b357037577bfa444e5e343e1b3b19020d14ef/pagefind/features/indexing.feature#L58-L84
